src/translations/fr/homepage.json

```
@@ -23,11 +23,14 @@
   "how.bloc4.title": "4. Publication",
   "how.title": "Fonctionnement",
   "mission_statement": "Quand les services que nous utilisons au quotidien changent leurs règles du jeu, nous sommes les premiers concernés.",
+  "reuses.memos-elections-fr.author": "Reset",
+  "reuses.memos-elections-fr.subtitle": "Suivi du comportement des plateformes sur des périodes électorales.",
```

All phrasings are in the active phrasing and the present tense.
```suggestion
  "reuses.memos-elections-fr.subtitle": "Suivi du comportement des plateformes sur des périodes électorales.",
```
```suggestion
  "reuses.memos-elections-fr.subtitle": "Suivi du comportement des plateformes sur des périodes électorales.",
```
```suggestion
  "reuses.memos-elections-fr.subtitle": "Suivi du comportement des plateformes sur des périodes électorales.",
```
```suggestion
  "reuses.memos-elections-fr.subtitle": "Suit le comportement des plateformes en période électorale.",
```